### PR TITLE
Block Entities 

### PR DESCRIPTION
**Pull request by EOT3000**
- [ ] Block Entities
  - [x] General
  - [ ] More NBT
- [ ] Biome Colors

### CODE REVIEW

**Comment by RednedEpic** [comment]
Block entities and biome colors should be a separate PR. 'more data' and 'extra data' will have to be elaborated on before a determination can be made about if they should be split into separate PRs as well.

**Comment by EOT3000** [comment]
idk what extra data is, it was just in nukkit so...

**Comment by RednedEpic** [comment]
> idk what extra data is, it was just in nukkit so...

Maybe block rotations? If so, we got that done.

**Comment by LegacyGamerHD** [comment]
this doesnt work on 1.13 branch plz fix the chunks are loaded and Block Entities are not visable yet

**Comment by RednedEpic** [comment]
> this doesnt work on 1.13 branch plz fix the chunks are loaded and Block Entities are not visable yet

Well yes, this PR is still very much a work in progress.

File: connector/src/main/java/org/geysermc/connector/network/translators/BlockEntityUtils.java [hunk 1]
```diff
@@ -83,29 +85,21 @@ public static String getBedrockID(String java) {
...
             int z = ((Number) tag.getValue().get("z").getValue()).intValue();
 
             String id = BlockEntityUtils.getBedrockID((String) tag.get("id").getValue());
 
-            list.add(new IntTag("x", x));
-            list.add(new IntTag("y", y));
-            list.add(new IntTag("z", z));
-
-            list.add(new StringTag("id", id));
+            return CompoundTagBuilder.builder().intTag("x", x).intTag("y", y).intTag("z", z).stringTag("id" , id).build("");
```
Comment (SupremeMortal) [inline connector/src/main/java/org/geysermc/connector/network/translators/BlockEntityUtils.java]: `.buildRootTag()` has the same effect.

**Comment by RednedEpic** [comment]
Closing due to this being moved to the [`block-entities`](https://github.com/GeyserMC/Geyser/tree/block-entities) branch.

**Pull request closed without merging**